src/dynamic_graph/centroidal_force_qp_controller.cpp

```diff
@@ -47,6 +47,13 @@ void CentroidalForceQPController::initialize(int number_endeffectors,
         number_endeffectors, friction_coeff, qp_penalty_lin, qp_penalty_ang);
 }
 
+void CentroidalForceQPController::set_slack_weights(dynamicgraph::Vector& weights)
```

Comment:
I would have preferred a modification of the initialize function.
Having another setter seems unnecessary here.

Reply:
Okay, I will change it.

Reply:
Done.